packages/appstore/src/components/add-options/add-options.scss

```diff
@@ -0,0 +1,42 @@
+.add-options--desktop {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    max-width: 240px;
```

Comment:
base on the project's standard should we use rem for widths?

Reply:
As I checked in the `docs/Stylesheet/README.md` file, there is no standard for width, but in line 83 we have this:
-   `px` is used to define a fixed value such as for `box-shadow`, `border-radius`, and `border-width`.
so since this max-width is a fixed value, I used 'px' for that.

Reply:
Alright, cool